starter-code/iteration-1/main.ts

```diff
@@ -1,18 +1,35 @@
 // Iteration 1
 // 1. Create an array of strings
+let testArray:Array<string>=[];
+let number=0;
 
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+function addTask (task:string){
+    testArray.push(task);
+    number++;
+    console.log (task+"was added");
+} 
 
 // 3. Create a function to list all tasks, it must show in the console de task.
-
+function listAllTasks (task:string){
```

Comment:
a better implementation would have set the type of the function to void and used a foreach to iterate, like this:
`let listAllTasks = function (): void {
  this.tasks.forEach((task, index) => {
    console.log(task);
  });
}`